### DISCUSSION TITLE
Mandatory JDBC support for new database

### DISCUSSION THREAD

**Discussion by MichaelA-regatta** [dbeaver/dbeaver#20863] [DBeaver Development] [answered]
Hello,

I'm currently working on a new JDBC driver for an upcoming database and would greatly appreciate guidance on the essential/mandatory JDBC functions needed for integration with DBeaver. 

For example, are methods like prepareStatement or getTables mandatory? 

Also, could you point me to the code section where I can find the complete list of necessary functions needed to run the DBeaver application?

Thank you😊

**Comment by E1izabeth** [comment, marked as answer]
Hi, we don't have something like a list of methods we use. Usually we rely on the assumption that all JDBC interfaces are implemented.
